### DISCUSSION TITLE
How to add new debugger

### DISCUSSION THREAD

**Discussion by OpenRay** [eclipse-embed-cdt/eclipse-plugins#603] [Debug Plug-ins Q&A]
My company has developed a debugger（such as jLink or openocd）for RISCV.
We have tested the debugger extensively and confirmed that it's working as expected. 
I did not find valid information from CONTRIBUTING.md to allow or not allow adding new debugger.
Please tell me know if there are any specific procedures or requirements for contributing to this repository. 
Thanks.

**Comment by ilg-ul** [comment]
I suggest you first publish your debugger on a company server, with documentation and everything, and allow your clients to use it.

**Reply by OpenRay** [reply]
ok, thanks.
